### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by lhbonifacio**
Update README

### CODE REVIEW

File: generate_queries_openai.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def parse_good_bad(data):
 
     args = parser.parse_args()
 
-    openai.api_key = os.getenv('OPENAI_API_KEY')
+    openai.api_key = os.environ['API_KEY']
```
Comment (rodrigonogueira4) [inline generate_queries_openai.py:46]: `getenv` didn't work?

**Review by rodrigonogueira4 — APPROVED** [review]
Thanks a lot for improving the documentation!

**Comment by hugoabonizio** [comment]
@lhbonifacio is it ready to merge?

**Comment by lhbonifacio** [comment]
> @lhbonifacio is it ready to merge?
Yes, it is. Can you approve and merge, please?

**Comment by rodrigonogueira4** [comment]
Just nudging on this. BTW, it is a common practice that the creator of PR is the one who merges it.

**Pull request merged**